.github/workflows/devbuild.yml

```diff
@@ -10,7 +10,7 @@ jobs:
 
   standalone_buffer:
 
-    if: ${{ github.event_name != '' || (github.event_name == '' && github.repository_owner == 'solvcon') }}
+    if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && vars.MMGH_NIGHTLY == 'enable') }}
```

Comment:
Could you explain to me when you'd like to build the nightly? I don't understand the logic here.

Reply:
My apologies for missing the reference to the issue #286.  The fencing was updated so that a forked repository will not run the workflows nightly.

Without the fencing, a forked repository will waste a lot of Github Actions cycles to duplicate the scheduled nightly workflow runs in the upstream repository.  To avoid the waste, the forked repository should turn off the Github Actions, and thus does not execute any workflow when a developer pushes to it.

With the fencing, the nightly runs are skipped unless the repository variable `MMGH_NIGHTLY` is set to `enable` in the forked repository.  The variable has been set in the upstream repository https://github.com/solvcon/modmesh.

Reply:
@yungyuc  But I wonder why it also builds when the event is not "schedule"? I think it should be built only once a day.

Reply:
`schedule` is just one of the many values of [`github.event_name`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/contexts#github-context): https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows .  When the triggering event is not `schedule`, the workflow should just run.

Reply:
Okay, I think I got your intention now.